src/components/Nav.js

```diff
@@ -1,15 +1,15 @@
 import { Link } from 'react-router-dom';
+import './Nav.css';
 
 function Nav() {
   return (
-    <nav>
-      <h1>Bookstore</h1>
-      <ul>
-        <li>
-          <Link to="/books">Books</Link>
+    <nav className="nav-bar">
+      <ul className="nav-list">
+        <li className="active-link">
+          <Link to="/books">BOOKS</Link>
         </li>
-        <li>
-          <Link to="/categories">Categories</Link>
+        <li className="active-link">
+          <Link to="/categories">CATEGORIES</Link>
         </li>
```

Comment:
- [ ] Please the placeholder image at the top right of the design is missing, please kindly add it as your implementation needs to match the design. Checkout the screenshot below and kindly fix it.

![placeholderImage](https://user-images.githubusercontent.com/62849872/179253847-ad634015-c96f-4e4c-b57d-2dd3f3fac993.png)